Packages/Application/Neos.NeosIo.Event/Configuration/NodeTypes.Speaker.yaml

```diff
@@ -0,0 +1,57 @@
+###########################################
+# Speaker node type
+###########################################
+
+'Neos.NeosIo.Event:Speaker':
+  superTypes:
+    'Neos.NeosIo.Event:Person': true
+  constraints:
+    nodeTypes:
+      'TYPO3.Neos.NodeTypes:Page': false
+  ui:
+    label: 'Speaker'
+    icon: 'icon-user'
+    inspector:
+      groups:
+        'person':
+          label: 'Speaker properties'
+  properties:
+    'photo':
+      type: TYPO3\Media\Domain\Model\ImageInterface
+      ui:
+        label: 'Photo'
+        reloadIfChanged: true
+        inspector:
+          group: 'person'
+          position: 10
+    'gender':
+      type: 'string'
+      ui:
+        label: 'Gender'
+        reloadIfChanged: true
+        inspector:
+          group: 'person'
+          position: 15
+          editor: 'TYPO3.Neos/Inspector/Editors/SelectBoxEditor'
+          editorOptions:
+            values:
+              'female':
+                label: 'Female'
+              'male':
+                label: 'Male'
```

Comment:
can't we remove "photo" and "gender" as it is inherited from the base person anyways?